spec/decorators/book_decorator_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe BookDecorator do
+  subject(:decorated_object) { described_class.new(book) }
+
+  let(:book) { FactoryBot.create(:book, authors: authors, categories: categories) }
+  let(:authors) do
+    [
+      FactoryBot.create(:author, first_name: FFaker::Name.first_name, last_name: FFaker::Name.last_name),
+      FactoryBot.create(:author, first_name: FFaker::Name.first_name, last_name: FFaker::Name.last_name)
+    ]
+  end
+  let(:categories) do
+    [
+      FactoryBot.create(:category, name: FFaker::Book.genre),
+      FactoryBot.create(:category, name: FFaker::Book.genre)
+    ]
+  end
+
+  context 'when decorate' do
```

Comment:
Where is context `'when not decorate'`? Rewrite decorator tests in order like in `spec/decorators/author_decorator_spec.rb`